tests/test_butlerFits.py

```diff
@@ -431,6 +432,45 @@ def testExposureFormatterAmpParameter(self):
             self.assertImagesEqual(test_t3_untrimmed.image, untrimmed_full[amp.getRawBBox()].image)
             self.assertAmplifiersEqual(test_t3_trimmed.getDetector()[0], amp_t3)
 
+    def testMaskedImageFormatter(self):
+        """Test that a MaskedImage can be persisted and read from a Butler."""
+        # Read in an Exposure as MaskedImage using MaskedImageFitsReader.
+        reader = MaskedImageFitsReader(os.path.join(TESTDIR, "data", "calexp.fits"))
+        mi = reader.read()
+
+        # Put the MaskedImage into the Butler and get a reference to it.
+        dataId = {"visit": 42, "instrument": "DummyCam", "physical_filter": "d-r"}
+        ref = self.butler.put(mi, "noise", dataId)
+        handle = self.butler.getDeferred(ref)
```

Comment:
Can you also do a butler.get(ref) to make sure you really do get the original masked image back?

Reply:
Added a test. I had initially avoided it thinking it checks a more generic butler capability and had nothing specific to do with `MaskedImage`.